internal/cmd/login_test.go

```diff
@@ -492,6 +491,8 @@ func TestLoginCmd_TLSVerify(t *testing.T) {
 		expected := &ClientConfig{ClientConfigVersion: clientConfigVersion}
 		assert.DeepEqual(t, cfg, expected, cmpopts.EquateEmpty())
 
-		golden.Assert(t, bufs.Stderr.String(), t.Name())
```

Comment:
It's not possible to compare the exact outputs since the server endpoint has a variable port component